drivers/ncv7356/ncv7356_trx.c

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2016  OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     ncv7356 trx
+ * @{
+ *
+ * @file
+ * @brief       generic can transceiver implementation for ncv7356
+ *
+ * @author      Aurelien GONCE <aurelien.gonce@altran.com>
+ * @author      Vincent Dupont <vincent@otakeys.com>
+ *
+ * @}
+ */
+
+#include "ncv7356_trx.h"
+
+int ncv7356_trx_set_mode(trx_t *dev, trx_mode_t mode)
+{
+    ncv7356_trx_t *ncv7356 = (ncv7356_trx_t *)dev;
+    int ret;
+
+    switch (mode) {
+    case TRX_NORMAL_MODE:
+    case TRX_SILENT_MODE:
+        gpio_set(ncv7356->mode0_pin);
+        gpio_set(ncv7356->mode1_pin);
+        ret = 0;
+        break;
+
+    case TRX_HIGH_SPEED_MODE:
+        gpio_set(ncv7356->mode0_pin);
+        gpio_clear(ncv7356->mode1_pin);
+        ret = 0;
+        break;
+
+    /*
+    case TRX_HIGH_VOLTAGE_WAKE_UP_MODE:
+        gpio_clear(ncv7356->mode0_pin);
+        gpio_set(ncv7356->mode1_pin);
+        ret = 0;
+        break;
+    */
```

Comment:
I highly discourage to commit any dead code - but at least it should be documented why this block is disabled.